pynest/nest/lib/hl_api_simulation.py

```diff
@@ -139,8 +144,20 @@ def RunManager():
     ::
 
         with RunManager():
-            for i in range(10):
-                Run()
+            for _ in range(10):
+                Run(100)
+                # extract results
+
+    Notes
+    -----
+
+    ```suggestion
```

Comment:
remove

Reply:
@tfardet Thanks for catching this, fixed. I decided to remove the bold everywhere, one tends to overuse it ;).